src/connector/postgres.rs

```diff
@@ -547,20 +551,6 @@ impl PostgreSql {
             }
         }));
 
-        // SET NAMES sets the client text encoding. It needs to be explicitly set for automatic
-        // conversion to and from UTF-8 to happen server-side.
-        //
-        // Relevant docs: https://www.postgresql.org/docs/current/multibyte.html
-        let session_variables = format!(
-            r##"
-            {set_search_path}
-            SET NAMES 'UTF8';
```

Comment:
`rust-postgres` already sets that to `UTF8`, as you can see here https://github.com/prisma/rust-postgres/blob/614e66c539c6127a8749225d4a6f7c67fe21c0bb/tokio-postgres/src/connect_raw.rs#L120.

If you're unsure whether it is referring to the same setting, [have a look here](https://www.postgresql.org/docs/current/multibyte.html#:~:text=Using%20the%20configuration%20variable%20client_encoding.%20If%20the%20client_encoding%20variable%20is%20set%2C%20that%20client%20encoding%20is%20automatically%20selected%20when%20a%20connection%20to%20the%20server%20is%20made.%20(This%20can%20subsequently%20be%20overridden%20using%20any%20of%20the%20other%20methods%20mentioned%20above.))